### PR TITLE
Chore/reset database

### PR DESCRIPTION
**Pull request by rachel-bock**
What does this PR do?
- Update 4 photos to remove white border
- Reset the trees table of the database to separate out the attributes into separate fields and add grows_in_zones field

Where should your reviewer start?
- init.sql 
- trees-dbv6.sql

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- N/A

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

**Review by irmakerdem — APPROVED** [review]
Thanks for editing the white frames out!

**Review by Baskanbetul — APPROVED** [review]
Thank you for resizing images and adding.  🤞🏼

File: init.sql [hunk 1]
```diff
@@ -152,502 +152,84 @@ INSERT INTO zip_codes (id, sid, default_state, city, default_type, zip_code, gro
...
...lliance Serviceberry','Amelanchier x grandiflora ''Autumn Brilliance''','Native Tree with Four Seasons of Interest','Superb Size for Smaller Lots','Pink Buds Open to Masses of Lacy, White Flowers','Butterflies and Beneficial Pollinators Adore the Delicate Blooms','Wonderful Food Source for Birds','Native Disease-Resistance','3-9','Full Sun, Partial Shade','Well Drained','20 - 25 feet','15 - 20 feet','Medium','White','Blue-Green','Orange, Red','Yes','https://www.naturehills.com/autumn-brilliance-serviceberry','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-multi-stemmed.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-blooms.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-full-600x600.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-4-800x800.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/autumn-brilliance/autumn-brilliance-serviceberry-fall-color.webp');
```
Comment (BrandonAinsworth) [inline init.sql:235]: Awesome job reseeding the database! Trees sometimes struggle to grow without proper seeding 😁

File: trees-dbv6.sql [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+INSERT INTO trees(ID,growing_zone,tree_name,botanical_name,descrip1,descrip2,descrip3,descrip4,descrip5,descrip6,grows_in_zones,sun_preference,soil_preference,mature_height,mature_spread,growth_speed,flower_color,foliage_color,fall_color,pollinator_friendly,outgoing_link,main_tree_image,tree_image_1,tree_image_2,tree_image_3,tree_image_4) VALUES (16,7,'Colorado Spruce','Picea pungens','Wild Bird Interest','Hardy & Easy-Care','Graceful Strength','Tolerant & Resilient','Boughs For Floral Design','Deer Resistant','2-7','Full Sun, Partial Shade','Well Drained','50 - 60 Feet','12 - 20 Feet','Slow','N/A','Green, Blue','Evergreen','No','https://www.naturehills.com/colorado-spruce','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/colorado-spruce/3327-2_piceapungenscoloradospruce-600x600_1.webp','https://raw.githubusercontent.com/irmakerdem/tree-pal-api/main/assets/colorado-spruce/3327-1_piceapungenscoloradospruce-600x600_1.webp','N/A','N/A','N/A');
```
Comment (BrandonAinsworth) [inline trees-dbv6.sql:44]: The new images look great! The uniformity will be pleasing to the user. 🌳

**Review by BrandonAinsworth — APPROVED** [review]

**Pull request merged**